src/main/java/Group15/MuscleSelectionView.java

```diff
@@ -0,0 +1,127 @@
+package Group15;
+
+import Group15.Api.BodyPart;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.*;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class MuscleSelectionView {
+
+    public static Scene createMuscleSelectorScene() {
+        VBox vBox = new VBox();
+        HBox inputAndEquipBox = new HBox();
+        HBox imageAndTextButtons;
+
+        List<String> selectedBodyparts = new ArrayList<>();
+
+        ComboBox equipmentSelector = new ComboBox<>();
+        equipmentSelector.getItems().addAll(
+                "All",
+                "Body weight",
+                "Barbell",
+                "Dumbbell",
+                "Machine"
+        );
+        equipmentSelector.setMinSize(200,50);
```

Comment:
I still cant choose more than one equipment

Reply:
Look at earlier comment, combobox doesnt allow for multiple selection out of the box

Reply:
Then you need to change that as well...